src/select/util/selectTypes.ts

```diff
@@ -1,13 +1,19 @@
 import React from "react";
 
-interface Option {
-  id: string;
+interface Option<Id = string> {
+  id: Id;
+  title: React.ReactNode;
```

Comment:
Why did we add `title` property here?

Reply:
I think it can be useful for some cases, but I can set it as optional or remove it completely.